README.md

```diff
@@ -1,14 +1,16 @@
 
 ![Hyperledger Sawtooth](images/sawtooth_logo_light_blue-small.png)  
 
-Hyperledger Sawtooth
--------------
+Sawtooth Supply Chain
+-----------------
+
+This is a distributed application to help you trace the provenance and other contextual information of any asset.
+ It can be used as-is or customized for different usages.
+This supply chain dApp runs on top of Hyperledger Sawtooth, an enterprise blockchain. 
```

Comment:
> dApp

Reply:
Not sure what your feedback is. Do you not like the term dApp? I know I don't, but there it is anyway.  :)